docs/source/reference/providers.rst

```diff
@@ -65,14 +56,14 @@ You can now create a Scaleway session:
 
 You can also create a Scaleway session using a ``with`` block:
 
->>> with scw.Session(platform=PLATFORM_NAME, project_id=PROJECT_ID, token=TOKEN) as session:
+>>> with scw.Session(platform_name=PLATFORM_NAME, project_id=PROJECT_ID, token=TOKEN) as session:
 ...     #
 ...     # Session scope
 ...     #
 
 Note: using a ``with`` block you do not need to start and stop your session: it starts automatically at the beginning of the block and stops automatically at its end.
 
-.. note:: while using a Jupyter Notebook for convenience python objects are kept alive and we recommend using directly ``start`` and ``stop`` methods.
+Note: while using a Jupyter Notebook for convenience python objects are kept alive and we recommend using directly ``start`` and ``stop`` methods.
```

Comment:
Same than before, the `.. note::` syntax generates a box titled "Note", which is visually better than just to have it in text format.
I would prefer to leave this line untouched.